src/Features/CSharpTest/ExtractMethod/ExtractLocalFunctionTests.cs

```diff
@@ -728,15 +727,12 @@ static void Outer(Action<int> x, int y)
 
                     static void Main()
                     {
-                        Outer(y => Inner(x =>
-                        {
-                            {|Rename:GetX|}(x).Ex();
+                        Outer(y => Inner(x => {|Rename:GetX|}(x).Ex(), y), (object)- -1);
 
-                            static string GetX(string x)
-                            {
-                                return x;
-                            }
-                        }, y), (object)- -1);
```

Comment:
small change in behavior.  When extracting out a local function we pick the highest local function we're in (if we find one) to go at the end of.